packages/combobox/README.md

```diff
@@ -87,7 +87,7 @@ import { Combobox, ComboboxOption } from '@leafygreen-ui/combobox';
 | `overflow`               | `'expand-y'`, `'expand-x'`, `'scroll-x'`      | Defines the overflow behavior of a multiselect combobox. **expand-y**: Combobox has fixed width, and additional selections will cause the element to grow in the block direction. **expand-x**: Combobox has fixed height, and additional selections will cause the elemenet to grow in the inline direction. **scroll-x**: Combobox has fixed height and width, and additional selections will cause the element to be scrollable in the x (horizontal) direction. | `'expand-y'`             |
 | `label`                  | `string`                                      | An accessible label for the input, rendered in a `<label>` to the DOM                                                                                                                                                                                                                                                                                                                                                                                               |                          |
```

Comment:
Should the same be done for `label`?

Reply:
Potentially - this ticket only asked for Description. But WDYT?

Reply:
Let's create a ticket for the labels to track it. And if it's not too much, we can get it done in this PR

Reply:
Will create a ticket - I think this will actually bea good thing to tackle as a part of CCP.